colossus/src/main/scala/colossus/protocols/http/HttpClient.scala

```diff
@@ -2,14 +2,20 @@ package colossus
 package protocols.http
 import scala.language.higherKinds
 
-import core._
 import service._
 import scala.concurrent.{ExecutionContext, Future}
 
-trait HttpClient[M[_]] extends LiftedClient[Http, M] {
+trait HttpClient[M[_]] extends LiftedClient[Http, M] with HttpRequestBuilder[M[HttpResponse]]{
+
+  protected def build(req: HttpRequest) = client.send(req)
```

Comment:
This returns an M[HttpResponse]?  Build feels like a bit of a misnomer, since it is actually invoking the client and sending the request.